### DISCUSSION TITLE
Shortcuts for volume

### DISCUSSION THREAD

**Discussion by Ramatonic** [SafeExamBrowser/seb-win-refactoring#422] [Q&A] [answered]
Hello,
I am configuring SEB for a blind student, which use a software to read the screen.
I have enabled the SEB volume icon in SEB taskbar, but I'd like to know if there are some SEB keyboard shortcuts to turn the volume up and down.
Other keyboard shortcuts seems to be not available: for example, on my laptop I have F7 and F8 to do that, but even if I enable them in SEB configuration they have no effect while SEB is running.

Thanks in advance

**Comment by dbuechel** [comment, marked as answer]
There currently are no shortcuts which SEB itself would provide. Some hardware volume keys may work, though a quick test showed that e.g. function keys do not necessarily work. You may have more luck when using the kiosk mode "Disable Explorer Shell".

There are major improvements in the works regarding accessibility, which should allow blind students to navigate the SEB shell with a keyboard in conjunction with e.g. a screen reader. These changes will be released with the upcoming version 3.4.0.

**Reply by Ramatonic** [reply]
Thanks for the detailed answer.
